service/application/src/main/resources/application.yml

```diff
@@ -5,6 +5,19 @@ springdoc:
   swagger-ui:
     path: /event-management-agent/swagger-ui.html
 
+logging:
+  log-in-json-format: false
+  level:
+    root: INFO
+  file:
+    name: /tmp/EMA.log
+  logback:
+    rolling-policy:
+      file-name-pattern: '${LOG_FILE}.%d{yyyy-MM-dd}.%i.gz'
+      max-file-size: 10MB
+      max-history: 7
+      total-size-cap: 1GB
+
```

Comment:
New log related configurations

Reply:
is `clean-history-on-start: false` missing here?

Reply:
It was not required : https://sol-jira.atlassian.net/browse/DATAGO-77753?focusedCommentId=1139242
Also, EMA does not keep any state so deleting the container will also delete the logs on every deployment 
> Please ensure that the application is compatible with this configuration (i.e. the logs are saved to the file /opt/ema/shared-logs/ema.log (assuming the directory /opt/ema/shared-logs exists (in K8S, we will be mounting this as en empty directory to the container)) and are emitted in JSON format).